src/translations/en.yml

```diff
@@ -0,0 +1,20 @@
+full_page:
+  body: '
+      Against the destruction of our social model.<br />
+      Against the constant growth of inequality.<br />
+      Against the systematic stigmatization of minorities.<br />
+      <span class="dcs-website-name__default">This</span>
+      <span class="dcs-website-name__prefix">The</span>
+      <span class="dcs-website-name__website">website</span>
+      <span class="dcs-website-name"></span>
+      is on strike.'
+  ready:  'Read the call ➔'
```

Comment:
I'm not sure "call" is correct here.
What do you think about "our manifesto" instead?

Reply:
Go for manifesto, that's better :)